### PR TITLE
Feature/dark theme

### PR DESCRIPTION
**Pull request by Epsilon02**

### CODE REVIEW

File: src/app/@core/models/global.model.ts [hunk 1]
```diff
@@ -20,12 +20,12 @@ export class GlobalModel {
 
   substanceData: SubstanceData[] = [];
 
-  humanAndEnvironmentDanger: ListInputSpecifcations[] = [];
+  humanAndEnvironmentDanger: string[] = [];
```
Comment (6543) [inline src/app/@core/models/global.model.ts:23]: looks like unrelated changes?

Comment (Epsilon02) [reply src/app/@core/models/global.model.ts]: Is fixed, checked this branch accidentally from another feature branch

File: src/app/menubar/menubar.component.ts [hunk 2]
```diff
@@ -13,14 +13,16 @@ import logger, { log } from '../@core/utils/logger';
...
 
-  searchResults: string[] = ['foo', 'bar', 'baz'];
+  descriptions = descriptions;
 
   constructor(
     public globals: GlobalModel,
     private loadSaveService: LoadSaveService
-  ) {}
+  ) {
+  }
```
Comment (6543) [inline src/app/menubar/menubar.component.ts:25]:
```suggestion
  ) {}
```

**Review by 6543 — APPROVED** [review]
